templates/download/desktop/index.html

```diff
@@ -34,6 +34,26 @@ <h2>Ubuntu {{lts_release_with_point}} LTS</h2>
         </div>
       </div>
     </div>
+    <div class="row">
+      <div class="col-12 p-card--highlighted">
+        <h2>Ubuntu {{ latest_release_with_point }}</h2>
+        <div class="u-equal-height p-divider">
+          <div class="col-8 p-divider__block">
+            <p class="p-card__content">The latest version of the Ubuntu operating system for desktop PCs and laptops, Ubuntu {{ latest_release_with_point }} comes with nine months, until {{latest_release_eol}}, of security and maintenance updates.</p>
+            <p class="p-card__content"><a href="https://wiki.ubuntu.com/{{latest_release_name}}/ReleaseNotes" class="p-link--external">Ubuntu {{ latest_release }} release notes</a></p>
+            <p class="p-card__content">Recommended system requirements are the same as for Ubuntu {{lts_release_full_with_point}}.</p>
+          </div>
+          <div class="col-4">
+            <p class="p-card__content">
+              <a class="p-button--positive is-wide js-latest-download" href="/download/desktop/thank-you/?version={{ latest_release_with_point }}&amp;architecture=amd64" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Download', 'eventAction' : 'Desktop', 'eventLabel' : '{{ latest_release }}', 'eventValue' : undefined });">Download</a>
+            </p>
+            <p class="p-card__content">
+              <a href="/download/alternative-downloads">Alternative downloads and torrents&nbsp;&rsaquo;</a>
```

Comment:
for 18.04 this is wrapped in a small tag

Reply:
Done, thanks